strongbox-web-forms/src/main/java/org/carlspring/strongbox/forms/configuration/MavenRepositoryConfigurationForm.java

```diff
@@ -1,13 +1,11 @@
 package org.carlspring.strongbox.forms.configuration;
 
-import org.carlspring.strongbox.providers.layout.Maven2LayoutProvider;
-
 import com.fasterxml.jackson.annotation.JsonTypeName;
 
 /**
  * @author Przemyslaw Fusik
  */
-@JsonTypeName(Maven2LayoutProvider.ALIAS)
+@JsonTypeName("Maven 2")
```

Comment:
Why not use the constant...?

Reply:
because we will have to be dependent on every layout provider. Should it be ?